src/components/m-button/mixins/_button.scss

```diff
@@ -84,7 +84,7 @@
   &:disabled,
   &[aria-disabled] {
     /* stylelint-disable declaration-no-important */
-    color: $color-button-text !important;
+    color: $color-prim-gray-dusty !important;
```

Comment:
Please use a distinct variable like `$color-button-text--disabled`, define it's default value in the button settings scss file.